packages/tadviewer/src/components/TadViewerPane.tsx

```diff
@@ -15,13 +15,20 @@ import { initAppState } from "../actions";
 import { AppState } from "../AppState";
 import { PivotRequester } from "../PivotRequester";
 import { actions } from "../tadviewer";
-import { AppPaneBaseProps, AppPane, tadReact } from "./AppPane";
+import { AppPane, AppPaneBaseProps } from "./AppPane";
+
+export interface CellClickData {
+  readonly value: string;
+  readonly cell: { row: number; col: number };
```

Comment:
Either make the sub-properties (`row`, `col`, `displayName`, and `columnType`) readonly as well, or flatten the type.